mitmproxy/tools/console/defaultkeys.py

```diff
@@ -7,12 +7,13 @@ def map(km):
     km.add("K", "console.view.keybindings", ["global"], "View key bindings")
     km.add("O", "console.view.options", ["global"], "View options")
     km.add("E", "console.view.eventlog", ["global"], "View event log")
+    km.add("2", "console.view.http2", ["global"], "View the HTTP/2 flow")
```

Comment:
I'm not sure if "flow" is an intuitive description here? From what I understand, `consolve.view.http2` displays exactly one HTTP/2 connection (from a single TCP socket) and shows all frames.
If this is correct, we should just tell the user that it is a "connection", not an abstract-sounding "flow".